### PR TITLE
change the config from hypen to camel casing format for yaml

### PR DESCRIPTION
**Pull request by hemanthgk10**
Signed-off-by: Hemanth Gokavarapu <hemanth.gokavarapu@lacework.net>

## Jira
https://lacework.atlassian.net/browse/COD-3829

## Description
- Changing the format of the local configuration from hypen to yaml and moving them under scan option.

### CODE REVIEW

File: .lacework/codesec.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 default:
   sca:
-    enable-dynamic: true
-    enable-fast: true
+    scan:
```
Comment (jeremydubreil) [inline .lacework/codesec.yaml:3]: Actually, we can probably delete this file instead of changing the format.

Comment (hemanthgk10) [reply .lacework/codesec.yaml:3]: this file is not being used?

Comment (jeremydubreil) [reply .lacework/codesec.yaml:3]: They are used for the e2e tests. What happens if you delete this file?

Comment (hemanthgk10) [reply .lacework/codesec.yaml:3]: I don't know much about the repo, I can delete and try.

Comment (hemanthgk10) [reply .lacework/codesec.yaml:3]: seems like it is not for the tests but the repo uses it to run those checks in github action.

Comment (hemanthgk10) [reply .lacework/codesec.yaml:3]: I see that you added it.

https://github.com/lacework/code-security-action/commit/7bf79db5c2970cbf288c77dc9ffd467b33ed6445

Comment (jeremydubreil) [reply .lacework/codesec.yaml:3]: Yes, I did but back then FAST was not enabled by default. I do not know about dynamic, maybe we need to keep this one. Are the corresponding triggered by these changes?

**Pull request closed without merging**